fuzz/ossfuzz/build.sh

```
@@ -50,6 +50,13 @@ build_fuzzer "assimp_roundtrip_fuzzer" "../fuzz/assimp_roundtrip_fuzzer.cc"
 cp ../fuzz/assimp_fuzzer.dict $OUT/assimp_roundtrip_fuzzer.dict || true
 
 
+# 1c. Post-processing Fuzzer (import any format, then apply the post-processing
+#     steps not exercised by the generic/round-trip fuzzers).
+build_fuzzer "assimp_postprocess_fuzzer" "../fuzz/assimp_postprocess_fuzzer.cc"
+(cd ../test/models && zip -q -r $OUT/assimp_postprocess_fuzzer_seed_corpus.zip .)
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)